packages/gestalt-codemods/34.0.0/__testfixtures__/ternary-color.input.js

```diff
@@ -3,6 +3,7 @@ import { Toast } from 'gestalt';
 
 export default function TestComponent() {
   return (
+    // eslint-disable-next-line no-constant-condition
```

Comment:
@dangerismycat should we make these `error` so it fails the build instead of silently going in?

Reply:
I'm not sure I follow?

Reply:
@dangerismycat `no-constant-condition` is set as a warning now, so it didn't fail the CI build (which is what we want).

![image](https://user-images.githubusercontent.com/127199/132867630-99452fe7-23c6-4870-bb02-e33df637fddc.png)

2 potential fixes:

* Set `--max-warnings` to `0` https://eslint.org/docs/user-guide/command-line-interface
* Convert the warning to an `error` in our `.eslintrc.json` file:
   ```json
   "rules": {
       "no-constant-condition": "error",
   ```

I recommend the second option since it works way better with IDE extensions

Reply:
https://github.com/pinterest/gestalt/pull/1677#discussion_r704700926

Reply:
Can we just set the true in a const in scope? (That was my initial suggestion)

Reply:
> Can we just set the true in a const in scope? (That was my initial suggestion)

That wouldn't fix this — we're still hardcoding something to a boolean, which is what this rule is designed to catch